MdeModulePkg/Core/Pei/Ppi/Ppi.c

```diff
@@ -324,16 +324,20 @@ ConvertPpiPointersFv (
 
     Guid = PrivateData->PpiData.PpiList.PpiPtrs[Index].Ppi->Guid;
     for (GuidIndex = 0; GuidIndex < ARRAY_SIZE (GuidCheckList); ++GuidIndex) {
+      // MU_CHANGE Start - CodeQL Change
       //
       // Don't use CompareGuid function here for performance reasons.

```

Comment:
This explicitly says don't use CompareGuid, are we sure we want to?

Reply:
There was discussion around this when the codeql changes were first merged. 
Based on the discussion back then, this was an artifact, and we decided to just use the CompareGuid to satisfy codeql.